README.md

```diff
@@ -59,8 +59,28 @@ This plugin gets configuration information from the following structure in ~/.do
 [dockstore-file-s3cmd-plugin]
 client = /usr/bin/s3cmd
 config-file-location = /home/user/.s3cfg
+verbosity = Normal
 ```
 
 Set the client location to your own s3cmd client and also make sure that the configuration file is available at the config-file-location.
+Verbosity configuration lets you configure the level of info display.  The default is level 2.
 
+### Verbosity
+- Level "Minimal" doesn't print the `s3cmd mb` stdout/stderr
+- Level "Normal" and higher prints the `s3cmd mb` stdout/stderr
```

Comment:
What are the options higher than "Normal"?

Reply:
Detailed and Diagnostics, but they are combined and only visible using the --debug flag instead.

Reply:
I don't mean to explain here, please explain in the README `Level "Normal" and higher ...`